language/diem-framework/modules/NetworkIdentity.move

```diff
@@ -0,0 +1,91 @@
+address 0x1 {
+/// Module managing Diemnet NetworkIdentity
+module NetworkIdentity {
+    use 0x1::DiemTimestamp;
+    use 0x1::Event::{Self, EventHandle};
+    use 0x1::Signer;
+    use 0x1::Vector;
+
+    /// An updatable `address` list with update notifications
+    struct AccountList has key {
+        accounts: vector<address>,
+        account_change_events: EventHandle<AccountListChangeNotification>
+    }
+
+    /// Message sent when there are updates to the `AccountList`.
+    struct AccountListChangeNotification has drop, store {
+        /// The new accounts
+        new_accounts: vector<address>,
+        /// The time at which the `accounts` was rotated
+        time_rotated_seconds: u64,
+    }
+
+    /// Holder for all `NetworkIdentity` in an account
+    struct NetworkIdentity has key {
+        identities: vector<u8>,
```

Comment:
Similar question here, on if we can bound this above by some number on-chain.

Reply:
Should this be a `vector<vector<u8>>` (if it is intended to be multiple serialized `NetworkIdentity` payloads)?

Reply:
Given right now that there is not full support for `vector<vector<u8>>` in the transaction-builder-generator, I'll be leaving it as a `vector<u8>`